### PR TITLE
FR - Third Parties Chapter

### PR DESCRIPTION
**Pull request by borisschapira**
Makes progress on #539

FR Translations of the third-parties chapter
also includes an update of the base template, because it has been modified in EN.

@AymenLoukil @allemas @nico3333fr @SilentJMA, it's open for review (feel free to use the out-of-box Github "Suggest Change" feature to propose modifications).

I took care to keep the original line numbers so if you checkout the branch locally, you can compare the original EN file with the FR file side by side.

### CODE REVIEW

File: src/content/fr/2019/third-parties.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+last_updated: 2019-12-21T00:00:00.000Z
+---
+
+## Introduction
+
+Le Web ouvert a été conçu pour être vaste, interconnectable et interopérable. La possibilité d’accéder à de puissantes librairies tierces et de les utiliser sur votre site avec des éléments `<link>` ou `<script>` a décuplé la productivité des développeurs et permis de nouvelles et incroyables expériences web ; par contre, l’immense popularité de quelques fournisseurs tiers pose d’importants problèmes en termes de performances et de confidentialité. Ce chapitre examine la prévalence et l’impact du code tiers sur le Web en 2019, les modèles d’utilisation du Web qui mènent à la popularité des solutions tierces et les répercussions potentielles sur l’avenir des performances Web et de la confidentialité.
+
+## Définitions
+
+### Ressource "tierce"
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
### "Tierce partie"
```

File: src/content/fr/2019/third-parties.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+Une partie tierce est une entité extérieure à la relation principale entre le site et l’utilisateur. Autrement dit, tous les aspects du site qui ne sont pas directement sous le contrôle du propriétaire du site mais qui sont présents avec son approbation. Par exemple, le script Google Analytics est un exemple de ressource tierce courante.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Une tierce partie est une entité extérieure à la relation principale entre le site et l’utilisateur. Autrement dit, tous les aspects du site qui ne sont pas directement sous le contrôle du propriétaire du site mais qui sont présents avec son approbation. Par exemple, le script Google Analytics est un exemple de ressource tierce courante.
```

File: src/content/fr/2019/third-parties.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+- Environ 84 % de tous les domaines tiers (en volume) ont été identifiés et catégorisés. Les 16 % restants appartiennent à la catégorie "Autres".
+
+## Données
+
+<figure>
+  <div class="big-number">93,59 %</div>
+  <figcaption>Figure 1. Pourcentage des pages destinées aux ordinateurs de bureau qui comprennent au moins une ressource tierce.</figcaption>
+</figure>
+
+Le code tiers est partout. 93 % des pages comprennent au moins une ressource tierce, 76 % des pages émettent une requête vers un domaine de télémétrie, une page requête, en médiane, du contenu en provenance d’au moins 9 domaines tiers _uniques_, ce qui représente 35 % de l’activité réseau totale. 10 % des pages, les plus "actives", émettent plus de 175 requêtes vers des ressources tierces. Il n’est pas exagéré de dire que les parties tierces font partie intégrante du Web.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Le code tiers est partout. 93 % des pages comprennent au moins une ressource tierce, 76 % des pages émettent une requête vers un domaine de télémétrie, une page requête, en médiane, du contenu en provenance d’au moins 9 domaines tiers _uniques_, ce qui représente 35 % de l’activité réseau totale. 10 % des pages, les plus "actives", émettent plus de 175 requêtes vers des ressources tierces. Il n’est pas exagéré de dire que les parties tierces font partie intégrante du web.
```

File: src/content/fr/2019/third-parties.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+<figure>
+  <div class="big-number">55,63 %</div>
+  <figcaption>Figure 2. Pourcentage de pages destinées aux ordinateurs de bureau qui comprennent au moins une ressource publicitaire.</figcaption>
+</figure>
+
+### Catégories
+
+Si l’omniprésence du contenu tiers n’est pas surprenante, la répartition du contenu de tiers par type de fournisseur est peut-être plus intéressante.
+
+Alors que la publicité pourrait être l’exemple le plus visible de contenus tiers sur le Web, les fournisseurs de services de télémétrie sont la catégorie de tiers la plus courante avec 76 % des sites incluant au moins une requête de ce type. Les CDN à 63 %, les annonces à 57 %, et les services comme Sentry, Stripe et Google Maps SDK à 56 % suivent de près en deuxième, troisième et quatrième position pour apparaître sur la plupart des plate-formes web. La popularité de ces catégories constitue le fondement de nos usage sur le Web, décrits plus loin dans ce chapitre.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Alors que la publicité pourrait être l’exemple le plus visible de contenus tiers sur le web, les fournisseurs de services de télémétrie sont la catégorie de tiers la plus courante avec 76 % des sites incluant au moins une requête de ce type. Les CDN à 63 %, les annonces à 57 %, et les services comme Sentry, Stripe et Google Maps SDK à 56 % suivent de près en deuxième, troisième et quatrième position pour apparaître sur la plupart des plate-formes web. La popularité de ces catégories constitue le fondement de nos usage sur le Web, décrits plus loin dans ce chapitre.
```

File: src/content/fr/2019/third-parties.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+### Fournisseurs
+
+Un nombre relativement restreint de fournisseurs dominent le paysage des services tiers : les 100 premiers domaines représentent 30 % des requêtes réseau sur le Web. Des moteurs comme Google, Facebook et YouTube sont en tête avec des pourcentages entiers de parts de marché chacun, mais des entités plus petites comme Wix et Shopify représentent aussi une partie substantielle de la popularité des tiers.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Un nombre relativement restreint de fournisseurs dominent le paysage des services tiers : les 100 premiers domaines représentent 30 % des requêtes réseau sur le web. Des moteurs comme Google, Facebook et YouTube sont en tête avec des pourcentages entiers de parts de marché chacun, mais des entités plus petites comme Wix et Shopify représentent aussi une partie substantielle de la popularité des tiers.
```

File: src/content/fr/2019/third-parties.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+
+<figcaption>Figure 4. Top 10 des requêtes les plus populaires.</figcaption>
+</figure>
+
+### Types de ressources
+
+La répartition du volume de contenu par type de ressource donne également un aperçu de la façon dont le code tiers est utilisé sur le Web. Alors que les requêtes sur le domaine principal sont composées à 56 % d’images, à 23 % de scripts, à 14 % de CSS, et seulement à 4 % de HTML, les requêtes vers des domaines tiers contienne davantage de scripts et de code HTML (32 % de scripts, 34 % d’images, 12 % HTML, et seulement 6 % de CSS). On pourrait penser que le code d’une partie tierce est moins souvent utilisé pour faciliter la conception et plus fréquemment pour faciliter ou observer les interactions que le code du domaine principal, mais une ventilation des types de ressources par type de partie tierce apporte de la nuance à cette idée. Alors que les CSS et les images sont majoritairement issus du domaine principal (respectivement 70 % et 64 %), les polices sont largement servies par des fournisseurs tiers, avec seulement 28 % provenant du domaie principal. Nous explorerons plus en détails ces usages plus loin dans ce chapitre.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
La répartition du volume de contenu par type de ressource donne également un aperçu de la façon dont le code tiers est utilisé sur le web. Alors que les requêtes sur le domaine principal sont composées à 56 % d’images, à 23 % de scripts, à 14 % de CSS, et seulement à 4 % de HTML, les requêtes vers des domaines tiers contienne davantage de scripts et de code HTML (32 % de scripts, 34 % d’images, 12 % HTML, et seulement 6 % de CSS). On pourrait penser que le code d’une partie tierce est moins souvent utilisé pour faciliter la conception et plus fréquemment pour faciliter ou observer les interactions que le code du domaine principal, mais une ventilation des types de ressources par type de partie tierce apporte de la nuance à cette idée. Alors que les CSS et les images sont majoritairement issus du domaine principal (respectivement 70 % et 64 %), les polices sont largement servies par des fournisseurs tiers, avec seulement 28 % provenant du domaie principal. Nous explorerons plus en détails ces usages plus loin dans ce chapitre.
```

File: src/content/fr/2019/third-parties.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+
+### Nombre de requêtes
+
+49 % de toutes les requêtes pointent vers des tiers. Avec 51 %, le domaine principal conserve garde donc la tête, puisqu’il héberge la moitié des ressources Web. Mais les sites qui utilisent des ressources tierces doivent le faire de manière importante car même si un peu moins de la moitié de toutes les requêtes proviennent de tiers, un petit nombre de pages n’en référencent pas du tout. En détails : aux 75e, 90e et 99e percentiles, la quasi-totalité de la page est constituée de contenu de tiers. En fait, pour certains sites s’appuyant fortement sur des plates-formes WYSIWYG distribuées comme Wix et SquareSpace, le document racine est parfois la seule requête sur le domaine principal !
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
49 % de toutes les requêtes pointent vers des tiers. Avec 51 %, le domaine principal conserve garde donc la tête, puisqu’il héberge la moitié des ressources web. Mais les sites qui utilisent des ressources tierces doivent le faire de manière importante car même si un peu moins de la moitié de toutes les requêtes proviennent de tiers, un petit nombre de pages n’en référencent pas du tout. En détails : aux 75e, 90e et 99e percentiles, la quasi-totalité de la page est constituée de contenu de tiers. En fait, pour certains sites s’appuyant fortement sur des plates-formes WYSIWYG distribuées comme Wix et SquareSpace, le document racine est parfois la seule requête sur le domaine principal !
```

File: src/content/fr/2019/third-parties.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+
+< !-- insert graphic of metric 05_11 -->
+
+Le nombre de demandes émises par chaque fournisseur tiers varie aussi considérablement selon la catégorie. Bien que les services de télémétrie soient la catégorie de tiers la plus répandue sur les sites Web, ils ne représentent que 7 % de toutes les requêtes réseau vers des tiers. Les publicités, en revanche, se trouvent sur environ 20 % de sites en moins, mais elles représentent 25 % de toutes les requêtes réseau vers des tiers. L’impact disproportionné de leurs ressources par rapport à leur popularité sera un thème que nous ne manquerons pas d’approfondir dans les données restantes.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Le nombre de demandes émises par chaque fournisseur tiers varie aussi considérablement selon la catégorie. Bien que les services de télémétrie soient la catégorie de tiers la plus répandue sur les sites web, ils ne représentent que 7 % de toutes les requêtes réseau vers des tiers. Les publicités, en revanche, se trouvent sur environ 20 % de sites en moins, mais elles représentent 25 % de toutes les requêtes réseau vers des tiers. L’impact disproportionné de leurs ressources par rapport à leur popularité sera un thème que nous ne manquerons pas d’approfondir dans les données restantes.
```

File: src/content/fr/2019/third-parties.md [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+
+### Poids en octets
+
+Bien que 49 % des requêtes proviennent de tiers, leur part sur le Web en termes d’octets est un peu plus faible, atteignant seulement 28 %. Il en va de même pour la répartition par types de ressources. Les polices d’écriture tierces représentent 72 % de toutes les polices d’écritures, mais seulement 53 % des octets correspondant aux polices d’écriture ; 74 % des requêtes concernent des documents HTML tiers, mais ils ne représentent que 39 % des octets HTML ; 68 % des requêtes vidéo viennent de tiers, mais ne représentent que 31 % des octets vidéo. Tout cela semble indiquer que les fournisseurs tiers sont des intendants responsables qui maintiennent la taille de leurs réponses à un niveau bas. Dans la plupart des cas, c’est effectivement. Enfin, jusqu’à ce que vous examiniez les scripts.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Bien que 49 % des requêtes proviennent de tiers, leur part sur le web en termes d’octets est un peu plus faible, atteignant seulement 28 %. Il en va de même pour la répartition par types de ressources. Les polices d’écriture tierces représentent 72 % de toutes les polices d’écritures, mais seulement 53 % des octets correspondant aux polices d’écriture ; 74 % des requêtes concernent des documents HTML tiers, mais ils ne représentent que 39 % des octets HTML ; 68 % des requêtes vidéo viennent de tiers, mais ne représentent que 31 % des octets vidéo. Tout cela semble indiquer que les fournisseurs tiers sont des intendants responsables qui maintiennent la taille de leurs réponses à un niveau bas. Dans la plupart des cas, c’est effectivement. Enfin, jusqu’à ce que vous examiniez les scripts.
```

File: src/content/fr/2019/third-parties.md [hunk 10]
```diff
@@ -0,0 +1,239 @@
...
+```<insert graphic of metric 05_12>```
+-->
+
+En ce qui concerne les fournisseurs tiers spécifiques, on trouve les mêmes poids-lourds en tête du classement du nombre de requêtes, qu’en tête du classement des poids en octets. Les seuls à ne pas respecter cette tendance sont les fournisseurs très médiatiques que sont YouTube, Shopify et Twitter, qui se hissent en tête des tableaux d’impact en poids.
+
+< !--```<insert table of metric 05_07>```-->
+
+### Exécution de scripts
+
+57 % du temps d’exécution des scripts provient de scripts tiers, et les 100 premiers domaines représentent déjà 48 % de la durée totale d’exécution des scripts sur le Web. Cela souligne l’impact considérable que quelques entités bien définies ont réellement sur les performances du Web. Ce sujet est étudié plus en profondeur dans la section [Répercussions > Performance](#performance).
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
57 % du temps d’exécution des scripts provient de scripts tiers, et les 100 premiers domaines représentent déjà 48 % de la durée totale d’exécution des scripts sur le web. Cela souligne l’impact considérable que quelques entités bien définies ont réellement sur les performances du web. Ce sujet est étudié plus en profondeur dans la section [Répercussions > Performance](#performance).
```

File: src/content/fr/2019/third-parties.md [hunk 11]
```diff
@@ -0,0 +1,239 @@
...
+
+< !--```<insert table of metric 05_08>```-->
+
+< !--```<insert table of metric 05_10>```-->
+
+Bien que l’on puisse dire beaucoup de choses sur la popularité et l’impact sur le rendement de chaque fournisseur, cette analyse plus subjective est laissée à la discrétion du lecteur et d’autres outils conçus à cette fin, comme le [third-party-web](https://thirdpartyweb.today).
+
+## Types d’utilisation
+
+Pourquoi les propriétaires de sites utilisent-ils un code tiers ? Comment ces contenus tiers peuvent-ils désormais représenter plus de la moitié des requêtes réseau ? Que font toutes ces requêtes ? Les réponses à ces questions se trouvent dans les trois principaux modèles d’utilisation des ressources tierces. En général, les propriétaires de sites font appel à des tiers pour générer et consommer les données de leurs utilisateurs, monétiser leurs expériences sur le site et simplifier le développement Web.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Pourquoi les propriétaires de sites utilisent-ils un code tiers ? Comment ces contenus tiers peuvent-ils désormais représenter plus de la moitié des requêtes réseau ? Que font toutes ces requêtes ? Les réponses à ces questions se trouvent dans les trois principaux modèles d’utilisation des ressources tierces. En général, les propriétaires de sites font appel à des tiers pour générer et consommer les données de leurs utilisateurs, monétiser leurs expériences sur le site et simplifier le développement web.
```

File: src/content/fr/2019/third-parties.md [hunk 12]
```diff
@@ -0,0 +1,239 @@
...
+
+### Générer et consommer des données
+
+La télémétrie est la catégorie de tiers la plus populaire sur le Web et pourtant, elle est peu visible pour l’utilisateur. Considérez le volume d’information en jeu pendant la durée de vie d’une visite sur le Web : le contexte de l’utilisateur·ice, le dispositif, le navigateur, la qualité de la connexion, la localisation, les interactions avec la page, la durée de la session, l’information selon laquelle il ou elle a déjà visité le site ou cette page, etc. sont générés en permanence. Il est difficile, encombrant et coûteux de maintenir des outils qui permettent d’entreposer, de normaliser et d’analyser des séries chronologiques de données de cette ampleur. Bien que rien n’exige catégoriquement que la télémétrie relève du domaine des fournisseurs tiers, l’attrait grandissant pour la compréhension de vos utilisateur·ice·s, la grande complexité de l’espace des problèmes et l’importance croissante accordée à la gestion respectueuse et responsable des données font naturellement apparaître la télémétrie comme un modèle d’utilisation tiers très répandu.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
La télémétrie est la catégorie de tiers la plus populaire sur le web et pourtant, elle est peu visible pour l’utilisateur. Considérez le volume d’information en jeu pendant la durée de vie d’une visite sur le web : le contexte de l’utilisateur·ice, le dispositif, le navigateur, la qualité de la connexion, la localisation, les interactions avec la page, la durée de la session, l’information selon laquelle il ou elle a déjà visité le site ou cette page, etc. sont générés en permanence. Il est difficile, encombrant et coûteux de maintenir des outils qui permettent d’entreposer, de normaliser et d’analyser des séries chronologiques de données de cette ampleur. Bien que rien n’exige catégoriquement que la télémétrie relève du domaine des fournisseurs tiers, l’attrait grandissant pour la compréhension de vos utilisateur·ice·s, la grande complexité de l’espace des problèmes et l’importance croissante accordée à la gestion respectueuse et responsable des données font naturellement apparaître la télémétrie comme un modèle d’utilisation tiers très répandu.
```

File: src/content/fr/2019/third-parties.md [hunk 13]
```diff
@@ -0,0 +1,239 @@
...
+
+### Simplifier le développement
+
+Plus que tout autre usage, les ressources tierces sont surtout utilisées pour simplifier l’expérience de développement Web. Il est possible que même des modes d’utilisation antérieurs s’inscrivent dans ce schéma. Qu’il s’agisse d’analyser le comportement des utilisateurs, de communiquer avec les annonceurs ou de personnaliser l’expérience utilisateur, les ressources tierces sont utilisées pour faciliter le développement initial du site.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Plus que tout autre usage, les ressources tierces sont surtout utilisées pour simplifier l’expérience de développement web. Il est possible que même des modes d’utilisation antérieurs s’inscrivent dans ce schéma. Qu’il s’agisse d’analyser le comportement des utilisateurs, de communiquer avec les annonceurs ou de personnaliser l’expérience utilisateur, les ressources tierces sont utilisées pour faciliter le développement initial du site.
```

File: src/content/fr/2019/third-parties.md [hunk 14]
```diff
@@ -0,0 +1,239 @@
...
... majorité du contenu d’un site et ont naturellement un impact plus important sur les performances que les autres catégories de tiers, ils servent également des sites presque entièrement statiques qui exigent, dans la plupart des cas, très peu de JavaScript, ce qui ne devrait pas justifier le volume de temps d’exécution des scripts. Les autres catégories qui nuisent aux performances ont cependant encore moins d’excuses. Elles jouent un rôle très limité sur chaque page sur laquelle elles apparaissent et pourtant elles prennent rapidement le dessus sur une majorité de ressources. Par exemple, le bouton " J’aime " de Facebook et les widgets des réseaux sociaux associés occupent un espace extraordinairement réduit sur l’écran et ne représentent qu’une fraction de la plupart des expériences Web, et pourtant l’impact médian sur les pages avec des tiers sociaux est de près de 20 % de leur temps d’exécution JavaScript total. La situation est similaire pour l’analyse - les bibliothèques de suivi ne contribuent pas directement à l’expérience utilisateur perçue, et pourtant l’impact du 90e percentile sur les pages avec des tiers d’analyse est de 44 % de leur temps d’exécution JavaScript total.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Les publicités, les fournisseurs de services de réseautage social, d’hébergement et certains fournisseurs de services de télémétrie exercent une influence des plus négatives sur la performance du web. Alors que les fournisseurs d’hébergement fournissent la majorité du contenu d’un site et ont naturellement un impact plus important sur les performances que les autres catégories de tiers, ils servent également des sites presque entièrement statiques qui exigent, dans la plupart des cas, très peu de JavaScript, ce qui ne devrait pas justifier le volume de temps d’exécution des scripts. Les autres catégories qui nuisent aux performances ont cependant encore moins d’excuses. Elles jouent un rôle très limité sur chaque page sur laquelle elles apparaissent et pourtant elles prennent rapidement le dessus sur une majorité de ressources. Par exemple, le bouton " J’aime " de Facebook et les widgets des réseaux sociaux associés occupent un espace extraordinairement réduit sur l’écran et ne représentent qu’une fraction de la plupart des expériences web, et pourtant l’impact médian sur les pages avec des tiers sociaux est de près de 20 % de leur temps d’exécution JavaScript total. La situation est similaire pour l’analyse - les bibliothèques de suivi ne contribuent pas directement à l’expérience utilisateur perçue, et pourtant l’impact du 90e percentile sur les pages avec des tiers d’analyse est de 44 % de leur temps d’exécution JavaScript total.
```

File: src/content/fr/2019/third-parties.md [hunk 15]
```diff
@@ -0,0 +1,239 @@
...
+
+Le bon côté de ce petit nombre d’entités jouissant d’une si grande part de marché est qu’un effort très limité et concentré peut avoir un impact énorme sur le Web dans son ensemble. L’amélioration des performances chez les quelques premiers hébergeurs peut améliorer de 2 à 3 % de _toutes_ les requêtes web.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Le bon côté de ce petit nombre d’entités jouissant d’une si grande part de marché est qu’un effort très limité et concentré peut avoir un impact énorme sur le web dans son ensemble. L’amélioration des performances chez les quelques premiers hébergeurs peut améliorer de 2 à 3 % de _toutes_ les requêtes web.
```

File: src/content/fr/2019/third-parties.md [hunk 16]
```diff
@@ -0,0 +1,239 @@
...
+
+### Confidentialité
+
+L’abondance des fournisseurs de services de télémétrie et la forte concentration de l’exécution des scripts posent deux problèmes majeurs de confidentialité pour les visiteurs des sites : les propriétaires de sites sont très nombreux à pister leurs utilisateurs avec des scripts tiers, permettant à une poignée d’entreprises de recevoir des informations sur une grande partie du trafic Web.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
L’abondance des fournisseurs de services de télémétrie et la forte concentration de l’exécution des scripts posent deux problèmes majeurs de confidentialité pour les visiteurs des sites : les propriétaires de sites sont très nombreux à pister leurs utilisateurs avec des scripts tiers, permettant à une poignée d’entreprises de recevoir des informations sur une grande partie du trafic web.
```

File: src/content/fr/2019/third-parties.md [hunk 17]
```diff
@@ -0,0 +1,239 @@
...
+
+L’intérêt des propriétaires de sites pour la compréhension et l’analyse du comportement des utilisateurs n’est pas malveillant en soi, mais le caractère généralisé et plutôt dissimulé du pistage sur le Web soulève des préoccupations légitimes. Les utilisateurs et utilisatrices, les entreprises et les autorités législatives en ont pris conscience  ces dernières années, aboutissant aux réglementations sur la protection de la vie privée telles que le [RGPD](https://fr.wikipedia.org/wiki/R%C3 %A8glement_g%C3 %A9n%C3 %A9ral_sur_la_protection_des_donn%C3 %A9es) en Europe et le [CCPA](https://en.wikipedia.org/wiki/California_Consumer_Privacy_Act) en Californie. Il est essentiel de s’assurer que les équipes de développement traitent les données des utilisateurs et utilisatrices de manière responsable, qu’elles les traitent avec respect et qu’elles soient transparentes quant aux données recueillies pour que la télémétrie demeure la catégorie de tiers la plus populaire et pour s’assurer que l’analyse du comportement des utilisateurs reste en symbiose avec l’amélioration de la valeur d’usage pour les utilisateurs et utilisatrices.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
L’intérêt des propriétaires de sites pour la compréhension et l’analyse du comportement des utilisateurs n’est pas malveillant en soi, mais le caractère généralisé et plutôt dissimulé du pistage sur le web soulève des préoccupations légitimes. Les utilisateurs et utilisatrices, les entreprises et les autorités législatives en ont pris conscience  ces dernières années, aboutissant aux réglementations sur la protection de la vie privée telles que le [RGPD](https://fr.wikipedia.org/wiki/R%C3 %A8glement_g%C3 %A9n%C3 %A9ral_sur_la_protection_des_donn%C3 %A9es) en Europe et le [CCPA](https://en.wikipedia.org/wiki/California_Consumer_Privacy_Act) en Californie. Il est essentiel de s’assurer que les équipes de développement traitent les données des utilisateurs et utilisatrices de manière responsable, qu’elles les traitent avec respect et qu’elles soient transparentes quant aux données recueillies pour que la télémétrie demeure la catégorie de tiers la plus populaire et pour s’assurer que l’analyse du comportement des utilisateurs reste en symbiose avec l’amélioration de la valeur d’usage pour les utilisateurs et utilisatrices.
```

File: src/content/fr/2019/third-parties.md [hunk 18]
```diff
@@ -0,0 +1,239 @@
...
+
+La forte concentration de l’exécution des scripts est excellente au regard de l’impact potentiel des futures améliorations de performance, mais moins excitante pour ses implications en termes de confidentialité. 29 % de _tous_ les temps d’exécution de scripts sur le Web proviennent uniquement de scripts sur des domaines appartenant à Google ou à Facebook. C’est un pourcentage très important du temps CPU qui est contrôlé par ces deux seules entités. Il est essentiel de s’assurer que les mêmes protections de la vie privée que celles dont bénéficient les fournisseurs de services de télémétrie sont également appliquées dans les autres catégories : services, publicités, réseaux sociaux et développement.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
La forte concentration de l’exécution des scripts est excellente au regard de l’impact potentiel des futures améliorations de performance, mais moins excitante pour ses implications en termes de confidentialité. 29 % de _tous_ les temps d’exécution de scripts sur le web proviennent uniquement de scripts sur des domaines appartenant à Google ou à Facebook. C’est un pourcentage très important du temps CPU qui est contrôlé par ces deux seules entités. Il est essentiel de s’assurer que les mêmes protections de la vie privée que celles dont bénéficient les fournisseurs de services de télémétrie sont également appliquées dans les autres catégories : services, publicités, réseaux sociaux et développement.
```

File: src/content/fr/2019/third-parties.md [hunk 19]
```diff
@@ -0,0 +1,239 @@
...
+
+## Conclusion
+
+Le contenu tiers est partout. Cela n’est guère surprenant ; le principe de base du web est de permettre l’interconnexion et la mise en relation. Dans ce chapitre, nous avons examiné les contenus de tiers en termes de ressources hébergées en dehors du domaine principal. Si nous avions inclus les contenus tiers auto-hébergés (par exemple, les bibliothèques open source communes hébergées sur le domaine principal), l’utilisation de contenus tiers aurait été encore plus importante !
+
+Bien que la [réutilisation dans les technologies informatiques](https://en.wikipedia.org/wiki/Code_reuse) soit généralement une pratique exemplaire, les tiers sur le Web introduisent des dépendances qui ont un impact considérable sur la performance, la confidentialité et la sécurité d’une page. L’auto-hébergement et la sélection minutieuse des fournisseurs peuvent grandement contribuer à atténuer ces effets.
```
Comment (JustinyAhin) [inline src/content/fr/2019/third-parties.md]:
```suggestion
Bien que la [réutilisation dans les technologies informatiques](https://en.wikipedia.org/wiki/Code_reuse) soit généralement une pratique exemplaire, les tiers sur le web introduisent des dépendances qui ont un impact considérable sur la performance, la confidentialité et la sécurité d’une page. L’auto-hébergement et la sélection minutieuse des fournisseurs peuvent grandement contribuer à atténuer ces effets.
```

**Comment by bazzadp** [comment]
> For more consistency with the EN version and inside the FR version, the world "web" can be used only in lower case, except where is it iuse in uppercase in the original document.

I'd agree with this, unless French treats this differently.

In English [there is some argument as to whether web (abbreviation of World Wide Web) and internet should be considered proper nouns and given and initial capital letter or not](https://en.wikipedia.org/wiki/Capitalization_of_Internet), but many style guides (including [the one used for the Almanac](https://developers.google.com/style/word-list#letter-w)) explicitly say it should be lower case.

**Comment by bazzadp** [comment]
BTW when making the same correction many, many times I find it's easier to add this as one comment and advise to make the same change elsewhere, and not do each one individually in the GitHub GUI.

Accepting each change individually is a little slow, adds a commit each time, and also means other changes can get lost (e.g. you changed `Ressource "tierce"` to `"Tierce partie"` which could easily get lost if Boris does decide to make the Web->web change in his local copy and submit as one extra commit).

So once you get into more than a handful of the same change, the suggested changes features is less useful IMHO. Though others may disagree!

**Comment by borisschapira** [comment]
@JustinyAhin Thanks for the feedback, it's perfect and I'm processing it. In the meantime, don't hesitate to push a PR on `master' to add yourself to [the contributors](https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/src/config/2019.json#L150).

**Comment by borisschapira** [comment]
I replaced "Web" by "web" and "partie(s) tierce(s)" by "tierce(s) partie(s)", which also impacts some templates I've updated. @JustinyAhin thank you for spotting these writing consistency issues and feel free to signal others.

**Comment by bazzadp** [comment]
> I replaced "Web" by "web" and "partie(s) tierce(s)" by "tierce(s) partie(s)", which also impacts some templates I've updated. @JustinyAhin thank you for spotting these writing consistency issues and feel free to signal others.

Agreed. I've also updated the "Common notes for writing consistency" inn #539 but feel free to edit if I got it wrong or you have more tips to add.

Also note I added this:

> Please include "Makes progress on #539" in all pull requests so a link is created from the PR to this issue.

Will also edit this PR's description to add this.

**Comment by bazzadp** [comment]
@JustinyAhin you happy to approve this now? If so will merge in.

**Comment by JustinyAhin** [comment]
> 
> 
> @JustinyAhin you happy to approve this now? If so will merge in once I hear feedback on [8362a6f](https://github.com/HTTPArchive/almanac.httparchive.org/commit/8362a6f2a91cd2adc640a7e2109c54aededa9372)

Yes, I'm ok with this.

**Comment by bazzadp** [comment]
Thanks both!

**Pull request merged**